pkg/prompts/prompts.go

```diff
@@ -917,27 +963,27 @@ func PromptChain(
 	prompter Prompter,
 	prompt string,
 	subnetNames []string,
-	includePChain bool,
-	includeXChain bool,
-	includeCChain bool,
-	avoidBlockchainName string,
+	avoidPChain bool,
+	avoidXChain bool,
+	avoidCChain bool,
+	avoidSubnet string,
```

Comment:
why Subnet instead of blockchain

Reply:
I am not sure where this is coming from. Let me roll it back.